app/src/main/res/layout/news_item.xml

```diff
@@ -8,43 +8,48 @@
 
     <TextView
         android:id="@+id/vUsername"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:text="User Example"
+        android:layout_marginStart="@dimen/news_margin_default"
+        android:layout_marginEnd="@dimen/news_margin_default"
+        android:layout_marginBottom="@dimen/news_margin_default"
+        android:text="Title Example"
```

Comment:
This should be a `tools:text...` if it's just for showing on the Android Studio preview.

Reply:
OMG!!! i didnt know about that... its awsome!!